### ISSUE TITLE
K type temperature range

### ISSUE THREAD

**Issue by charlespax** [PaxInstruments/t400-firmware#148]
The int32_t tempTypK() function in intypek_constant.h only process temperatures up to 730 C because the variable used is an int. We'll have to change something here to get the full range.

![screen shot 2015-02-16 at 14 41 31](https://cloud.githubusercontent.com/assets/281642/6207925/f73140ca-b5e9-11e4-9f3e-dc42ed53f6b9.png)

**Comment by charlespax** [comment]
Moving to READINGS.

**Comment by charlespax** [comment]
Now that the lookup table is in PROGMEM it appears that we can enable readings above 30382 uV.

All I did was uncomment the rest of the lookup table in typek_constant.h and change `#define TEMP_TYPE_K_LENGTH 101` to `#define TEMP_TYPE_K_LENGTH 165`.

I tested this the best I could, using a light to heat the end of a thermocouple. Here's the graph:
![screen shot 2015-02-23 at 02 24 26](https://cloud.githubusercontent.com/assets/281642/6319760/08966524-bb04-11e4-8e38-0865dce22957.png)


Note: I looked into this after implementing the code snippet I posted in the Temperatures below 17.6 issue (https://github.com/PaxInstruments/t400-firmware/issues/160). Not sure if one thing affects the other.
